src/test/java/com/epam/tc/hw2/ex1/ExerciseFirstTest.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+public class ExerciseFirstTest extends BrowserTestBase {
+    // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void test4ItemsHeader() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("HOME")).getText())
+                .isEqualTo("HOME");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("CONTACT FORM")).getText())
+                .isEqualTo("CONTACT FORM");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("SERVICE")).getText())
+                .isEqualTo("SERVICE");
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("METALS & COLORS")).getText())
+                .isEqualTo("METALS & COLORS");
+        softAssertions.assertAll();
+    }
+
+    // 6. Assert that there are 4 images on the Index Page and they are displayed
```

Comment:
What about 6 step? Also, please, keep in mind previous comment and perform all necessary assertions...